pyth-sdk/README.md

```diff
@@ -81,3 +81,49 @@ println!("0.1 BTC and 0.05 ETH are worth: ({} +- {}) x 10^{} USD",
 ```
 
 This operation can be useful for pricing, e.g., an LP token that is backed by two underlying currencies.
+
+## Liquidity-related Pricing Measures
+
+A variety of measures should be taken for protocols looking to mitigate liquidity-related risks. To limit deposits and borrows, protocols could simply derive their limit from the off-chain liquidity estimates and then store those values in contract state. No explicit interaction with the Pyth SDK would be needed in this simple case.
+
+To adjust the price at which collateral is valued based on the liquidity information, a protocol can combine the current Pyth price and their estimate of liquidity:
```

Comment:
The reader is not going to understand how the code snippet below incorporates liquidity. The code snippet looks like it computes a discount, but there's no variable called liquidity or anything like that.

I think you need to describe here how the user should get the final rate using average market depth (or some other liquidity estimate)

Reply:
I think you should probably explain the motivation for this function briefly as well (in the same way that price_basket above does)